.github/workflows/build.yml

```diff
@@ -143,6 +143,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: '11'
+      - uses: gradle/wrapper-validation-action@v1
+      - uses: gradle/gradle-build-action@v2
+        with:
+          cache-read-only: ${{ env.MAIN_BRANCH != 'true' }}
+      - name: Generate distributions
+        run: ./gradlew jsBrowserDistributionMkdocs --continue
```

Comment:
I worry that previously the mkdocs were running serially. By calling this Gradle command before calling mkdocs, the modules can hypothetically be run in parallel.

Reply:
Related issue, hopefully we can close it with this: #546

Reply:
hopefully. though I've seen some error messages in gradle saying that the task is not correctly configured, so there are a lot of cache misses.

hopefully in the future Jetbrains will fix the task so that the gradle cache is more effective.